### DISCUSSION TITLE
V0.8.6-beta(Web1.6.7.0)

### DISCUSSION THREAD

**Discussion by cloud-fs** [cloud-fs/cloud-fs.github.io#208] [Announcements]
V0.8.6-beta(Web1.6.7.0)

1. 改进和完善115网盘的跨网盘文件变更同步功能
2. 改进115网盘请求频度，缓解115网盘风控的风险
3. 为了兼容更多的设备，WebUI框架从.net 9回退到.net 7
4. 其它性能改进和bug修复

<hr /><em>This discussion was created from the release <a href='https://github.com/cloud-fs/cloud-fs.github.io/releases/tag/v0.8.6-beta'>V0.8.6-beta(Web1.6.7.0)</a>.</em>

**Comment by Marxpipiman** [comment]
謝謝大大更新，但這版本還是會封，我是不知道115是怎麼回事？架EMBY怎麼了嗎？刮削一下元數據就封，雖然之前有網友提過，只要將所有檔案放置在同一個資料夾就不會被封，我自已試過的確如此，但這樣檔案就比較難整理了，不知道有沒有更好的解決方法
